skyportal/handlers/api/candidate/scan_report.py

```diff
@@ -124,6 +124,21 @@ def post(self):
             if not saved_range:
                 return self.error("No saved candidates range provided")
 
+            # Check if this report already exists
+            existing_report = session.scalars(
+                ScanReport.select(session.user_or_token).where(
+                    ScanReport.groups.any(Group.id.in_(group_ids)),
```

Comment:
didn't know that `any` operator existing on relationships, that's interesting. I wonder what the resulting query looks like